### PR TITLE
Fi 873 add diagnostic report blurb

### PR DESCRIPTION
**Pull request by czh-orz**
This PR adds a comment in the about section for the diagnostic report sequences that says why we search by patient + category first. I also changed another part of the sequence description to show the resource instead of the test title.

**Submitter:**
- [X] This pull request describes why these changes were made
- [X] Internal ticket for this PR: https://oncprojectracking.healthit.gov/support/browse/FI-873
- [X] Internal ticket links to this PR
- [X] Internal ticket is properly labeled (Community/Program)
- [X] Internal ticket has a justification for its Community/Program label
- [X] Code diff has been reviewed for extraneous/missing code
-  Tests are included and test edge cases
- Tests/code quality metrics have been run locally and pass

**Reviewer 1:**

Name:
- [x] Code is maintainable and reusable, reuses existing code and infrastructure
      where appropriate, and accomplishes the task's purpose
- [ ] The tests appropriately test the new code, including edge cases
- [ ] You have tried to break the code

**Reviewer 2:**

Name:
- [ ] Code is maintainable and reusable, reuses existing code and infrastructure
      where appropriate, and accomplishes the task's purpose
- [ ] The tests appropriately test the new code, including edge cases
- [ ] You have tried to break the code

### CODE REVIEW

File: generator/uscore/templates/sequence.rb.erb [hunk 1]
```diff
@@ -47,7 +47,9 @@ module Inferno
         <%searches.select { |search_param| search_param[:expectation] == 'SHALL' && !search_param[:must_support_or_mandatory]}.each do |search| %>
           * <%=search[:names].join('+ ')%><%end%>
         <% end %>
-        
+        <% if resource == 'DiagnosticReport' %>
+        We search by patient + category before doing a search by just patient in order to differentiate the two Diagnostic Report profiles.
```
Comment (radamson) [inline generator/uscore/templates/sequence.rb.erb:51]:
```suggestion
        Inferno will search by patient + category before doing a search by patient only in order to differentiate the two Diagnostic Report profiles. 
```

Comment (czh-orz) [reply generator/uscore/templates/sequence.rb.erb]: Updated the wording
